### DISCUSSION TITLE
Looking for using paddle OCR for license plates

### DISCUSSION THREAD

**Discussion by livelove1987** [PaddlePaddle/PaddleOCR#13079] [General]
Hi i found this link to the model: https://github.com/PaddlePaddle/PaddleOCR/blob/main/applications/README_en.md#transportation i dont know chinese and im wondering how could i get access to that model and also if it will work for Europe license plate numbers

**Comment by GreatV** [comment]
No, you need to train your own model on European license plate numbers.
